app/controllers/instructor/photos_controller.rb

```diff
@@ -5,8 +5,8 @@ def new
   end
 
   def create
-    @photo = @course.photos.create(photo_params)
-    redirect_to instructor_course_path(@course)
```

Comment:
We need to reference `current_course` instead of `@course` since we never define `@course`.